crates/core/src/client/constants.rs

```diff
@@ -77,3 +81,23 @@ lazy_static! {
 lazy_static! {
     pub static ref KAKAROT_CLIENT_VERSION: String = format!("kakarot_{}", env!("CARGO_PKG_VERSION"));
 }
+
+// This module contains constants related to transactions used to calculate
+// Starknet gas price
+lazy_static! {
+    pub static ref ACCOUNT_ADDRESS: FieldElement =
+        FieldElement::from_hex_be("0x07142FbF6E8C9C07b079D47727C6D2ff49970203bfd5Bd6ED0D740e0f5a344E7").unwrap();
```

Comment:
Maybe we could add the codes of these "demo" contracts so we are just aware of their code?